<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_fournisseur</class>
 <widget class="QDialog" name="gestion_fournisseur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1071</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>fournisseur</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>421</width>
       <height>261</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numéro</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom soc</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Type">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Lieu">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>lieu</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type eq</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ID">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_num">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_mtype">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifier">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>lieu</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mnum">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom soc</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type eq</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mid">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mnom">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mlieu">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>numéro</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>supp</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_supprimer">
       <property name="geometry">
        <rect>
         <x>175</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimer">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>150</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Idantifiant</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QRadioButton" name="radioButton_CHtype">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>type</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercher">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>chercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>360</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>exporter</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_CHlieu">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>lieu</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_commandes">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>591</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chercher">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_CHid">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>commande</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>421</width>
       <height>251</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>ajouter</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_ajoutC">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom soc</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>quantité</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomC">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_typeC">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_quaC">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_idC">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>100</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID f</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dateC">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>modifier</string>
      </attribute>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID f</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_quaCm">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nomCm">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>type</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifierC">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>nom soc</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_idCm">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_typeCm">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>quantité</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dateCm">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>120</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>supprimer</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_suppC">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Idantifiant</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_suppC">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>confirmer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QTableView" name="tableView_fourn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>611</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_n">
     <property name="geometry">
      <rect>
       <x>566</x>
       <y>310</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_trier">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>320</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>trier</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_type">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>320</y>
       <width>97</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>type</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_id">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>320</y>
       <width>97</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>id</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>statistiques</string>
    </attribute>
    <widget class="QCustomPlot" name="plot" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>921</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>mail</string>
    </attribute>
    <widget class="QLineEdit" name="rcpt_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_109">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>message</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="msg_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>571</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="mail_pass_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_107">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>attachement here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browse">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>160</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_93">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>recipient</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_104">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>subject</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_108">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mail pass</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subject_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
